youtubeDownload/src/test/java/org/example/FileUtilsTest.java

```diff
@@ -0,0 +1,12 @@
+package org.example;
+
+import org.junit.*;
+
+public class FileUtilsTest {
+
+    @Test
+    public void testHasSolution() throws Exception {
+        Assert.assertSame("foo was found somehow", Boolean.FALSE, FileUtils.runScript("dir foo", "."));
```

Comment:
This is just to have some base for extra test.
`foo` denotes here an invalid file,